SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -47,6 +47,23 @@ bool ComponentFactory::FillSchemaObject(Schema_Object* ComponentObject, UObject*
 			const FRepLayoutCmd& Cmd = Changes.RepLayout.Cmds[HandleIterator.CmdIndex];
 			const FRepParentCmd& Parent = Changes.RepLayout.Parents[Cmd.ParentIndex];
 
+#if TRACE_LIB_ACTIVE
+			if (USpatialLatencyTracer* LatencyTracer = USpatialLatencyTracer::GetTracer(NetDriver))
+			{
+				TraceKey TraceId = LatencyTracer->GetTraceKey(Object, Cmd.Property); 
+				if (TraceId == USpatialLatencyTracer::InvalidTraceKey)
+				{
+					// Possibly sending a nested property? 
+					TraceId = LatencyTracer->GetTraceKey(Object, Parent.Property);
+				}
+				if (TraceId != USpatialLatencyTracer::InvalidTraceKey)
```

Comment:
Do we want logging if TraceId == USpatialLatencyTracer::InvalidTraceKey?

Reply:
Added! 99% of the time this is invalid but I added some trace to check that it doesn't happen twice for the same entity because only one can be sent at once